### ISSUE TITLE
Submission failed on device that uses EN for the first time

### ISSUE THREAD

**Issue by henrytao-me** [cds-snc/covid-shield-mobile#370]
## What? 
The device that uses EN for the first time doesn't have any TEKs. If the app submits empty array of TEKs to server, the server will return error which users shouldn't care about. 

## Suggestion
Check if TEKs is empty. If yes, skip submission for today. The flow continues as normal the next day.

**Comment by timarney** [comment]
On a fresh device where no exposure keys exists yet.

Submit a code ->agree 

Server returns
```
WARN[14883] no keys provided component=server path=/upload route=/upload uuid=21645a31-828c-4812-5df8-45d726ceaae1
```

Resulting in an error
<img src="https://user-images.githubusercontent.com/62242/86415015-b5879f00-bc93-11ea-9392-925d53de44bd.jpg" width="200">

After saying "ok" the "under" screen shows the upload your random codes button which will loop into same flow.

<img src="https://user-images.githubusercontent.com/62242/86415278-86bdf880-bc94-11ea-8ad5-94ed687fa065.jpg" width="200">

**Comment by Stephen-PM** [comment]
Expected behavior: To the user it should not matter if they have several days worth of keys, only keys from today, or no keys. After they enter their one time code and then agree to upload their keys they should be taken to the _Thank You_ screen. The next day, and on each subsequent day for 14 days, they should be prompted to upload new keys.

**Comment by smcmurtry** [comment]
I've investigated this and I believe #372 really did close this issue. Here is what I did:

1. Simulated a fresh device with no random ids, without the `if` statement added in #372:
![Screen Shot 2020-07-02 at 10 54 58 PM](https://user-images.githubusercontent.com/5498428/86433124-11afea80-bcb7-11ea-9809-69818292da88.png)
2. Submitted a code and observed the bug flow: submit 8-digit code -> share random ids -> error -> home screen displaying "upload random codes"
3. Simulated a fresh device with no random ids, with the `if` statement from #372:
![Screen Shot 2020-07-02 at 11 11 12 PM](https://user-images.githubusercontent.com/5498428/86433950-576db280-bcb9-11ea-8bad-f18b013d60f4.png)
4. Submitted a code and observed the desired flow: submit 8-digit code -> share random ids -> home screen displaying "thank you"
